rapid/workflow/api_controller.py

```diff
@@ -288,7 +288,8 @@ def delete_object(self, endpoint, _id):
 
     def reset_action_instance(self, _id):
         try:
-            if self.action_instance_service.reset_action_instance(_id, True):
+            _json = request.get_json(silent=True)
+            if self.action_instance_service.reset_action_instance(_id, _json['complete_reset'] if _json and 'complete_reset' in _json else False):
```

Comment:
This reads funny. I would have expected `_json.get('complete_reset', False)` or something of that ilk, however logically they are identical so I'm not going to be TOO stingy.